huntsman/camera/pyro.py

```diff
@@ -118,12 +119,18 @@ def connect(self):
 
         self._connected = True
         self._serial_number = uid
+        self.name = self._proxy.name
         self.model = self._proxy.model
         self._file_extension = self._proxy.file_extension
         self._readout_time = self._proxy.readout_time
         self._filter_type = self._proxy.filter_type
         self.logger.debug("{} connected".format(self))
 
+        if self._proxy._has_focuser:
```

Comment:
Should this be the private variable or public property? I'm confusing myself to how this is set up.
```suggestion
        if self._proxy.has_focuser:
```

Reply:
Public property. Of a private variable. `self._proxy` is the Pyro4 connection to the remote `CameraServer`. `CameraServer.has_focuser` is a public property because it's supposed to be accessible remotely.